frame/support/src/storage/mod.rs

```diff
@@ -1361,10 +1358,22 @@ where
 	}
 }
 
+/// Returns the storage prefix for a specific pallet and storage string.
```

Comment:
```suggestion
/// Returns the storage prefix for a specific pallet name and storage name.
///
/// The storage prefix is `concat(twox_128(pallet_name), twox_128(storage_name))`.
```

I don't know how to name stuff better, storage_prefix means both the prefix used by the storage in the pallet like in the attribute: `#[pallet::storage_prefix = "foo"]`, and the final storage prefix in the trie.

I think context is enough for now, but if someone has an idea about different names it would be great.